### DISCUSSION TITLE
Seeding of Database not working

### DISCUSSION THREAD

**Discussion by abhishek2b4u** [opencrvs/opencrvs-core#7646] [Technical Q&A]
Hi Team,

While seeding the database I am getting the error saying :
Parent location “Location/XX” not found for District XXXXX

Please help to point out where I can check the issue.
PS: I am using admin3Pcode so have changed the location level to 3 in handler.ts
Done: yarn db:clear:all and seed again but not helpful

**Comment by rikukissa** [comment]
Hi @abhishek2b4u, thank you for your question. Are you using the default [locations.csv](https://github.com/opencrvs/opencrvs-countryconfig/blob/develop/src/data-seeding/locations/source/locations.csv), health-facilities.csv and crvs-facilities.csv or have you made changes to these?

**Reply by abhishek2b4u** [reply]
Hi @rikukissa , 
Thanks for helping out
I am using modified csv according to the needs of mine.

**Reply by rikukissa** [reply]
My hunch is that it is to do with the partOf -> id links in the items your http://localhost:3040/locations endpoint provides. So in other words, for one of the locations, the id `partOf` field points to is not in the list as some other items `id`. 

<img width="537" alt="image" src="https://github.com/user-attachments/assets/d0e6f2d6-0161-4e0f-863f-0661fa112f23">

I suggest you test with a minimum number of locations and keep adding locations back until you find the culprit.

**Reply by abhishek2b4u** [reply]
@rikukissa, I am pointing my partOf to admin2Pcode and it is unique for each location, And the issue it seems is that it is having problem with each location. 
So my understanding is that there is some issue with architecture of the linking it to main location file. Let me know if you want to have look on the file itself..

**Reply by rikukissa** [reply]
Yea, please that would help a lot @abhishek2b4u

**Comment by rikukissa** [comment]
@abhishek2b4u thanks for sending me the reference files. Please check a row starting with `5R4NWHF9` in hearth-facilities and change `LocationCI060302` to `Location/CI060302` and report back. Hope this fixes the issue.

**Reply by abhishek2b4u** [reply]
@rikukissa, Thanks for taking out time for helping me out. I have applied the suggested changes but still the error is coming as earlier one.
ERROR:
`Parent location "Location/CI" not found for District Autonome D'Abidjan`

**Reply by abhishek2b4u** [reply]
Hi @rikukissa , Can you please help me out with the issue.

**Comment by euanmillar** [comment]
Hi @abhishek2b4u,

Thank you very much for your interest in OpenCRVS. We are really pleased that you reached out. From your thread, we understood that you have technical questions that perhaps need more dedicated support. 

In order to help you in the best way we need to learn a bit more about you and your requirements. Would you mind taking 10 minutes to answer the following questions? 

We prioritise each request depending on a number of factors. Please bear in mind that we tend to prioritise helping governments and system integrators working on live country implementations.

We do our best to respond to every query but at busy times you may need to wait a while. In the meantime, please check our [documentation](https://documentation.opencrvs.org/) again as it is exhaustive. Perhaps there is a solution there that may help you.

If you would like some dedicated support, can you email the answers to these questions to [team@opencrvs.org](mailto:team@opencrvs.org)?

Please tell us your full name and how you wish to be referred to.
Please tell us the name of the organisation or company you work for.
Are you working with a government or system implementing agency on an active country implementation? ... If so, which country’s implementation are you working on?

Which of these describes your project best?

- Testing OpenCRVS for business development, educational purposes or for personal interest
- A research project to evaluate, configure and test OpenCRVS as a possible solution for a government, an active brief or proposal. (If so, tell us about your project, brief, the client and timeline if you can.)
- Actively installing and configuring OpenCRVS Core for production use in a country.

Kind regards

**Comment by abhishek2b4u** [comment]
Hi,
  Please find the answers below

Please tell us your full name and how you wish to be referred to. >>
Abhishek
Please tell us the name of the organisation or company you work for.
Techstride Cloud
Are you working with a government or system implementing agency on an
active country implementation? ... If so, which country’s implementation
are you working on? > We are system implementing agency working for Côte
d’Ivoire country.

Let me know if you need more information.

Thanks,
Abhishek Singh
